config/grunt/custom-tasks/init-rc-release.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+/**
+ * helper function
+ * @param {array} 
+ * @returns {bool}
+ */
+
+if (!String.prototype.isInList) {
+	String.prototype.isInList = function() {
+	   let value = this.valueOf();
+	   for (let i = 0, l = arguments.length; i < l; i += 1) {
+		  if (arguments[i] === value) return true;
+	   }
+	   return false;
+	}
+ }
+
+
+
+
+
+module.exports = function( grunt ) {
+	grunt.registerMultiTask(
+		"init-rc-release",
+		"central entry point to set options in one place",
+		function() {
+            const options = this.options({
+                failOnError: true,
+                enableSvnDeploy: false,
+                githubOwner: '',
+                githubRepo: '',
+                alternativeBranch: 'CI-test',
+                alternativeBranchPush: false,
+                enableGithubRelease: false,
+                noBump: false,
+        });
+        
+
+        let owner;
+        let repo;
```

Comment:
Please rename `repo` to `repository` for clarity.

Reply:
repo is the name used for this in Github documentation,  for instance : https://developer.github.com/v3/issues/milestones/